Containers/nextcloud/activate-fulltextsearch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ "$FULLTEXTSEARCH_ENABLED" != yes ]; then
+    # Basically sleep for forever if fulltextsearch is not enabled
+    sleep 365d
+fi
+echo "Activating fulltextsearch..."
+php /var/www/html/occ fulltextsearch:live -q
+sleep 365d
```

Comment:
```suggestion
    sleep inf
fi
echo "Activating fulltextsearch..."
php /var/www/html/occ fulltextsearch:live -q
sleep inf
```
Why not directly sleep forever?
See: https://www.gnu.org/software/coreutils/manual/html_node/sleep-invocation.html

Reply:
good point! didn't know that this is possible...

Reply:
fixed